README.md

```diff
@@ -107,7 +107,6 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 The ActiveRecord failover tests are run against a dummy Rails server. Run the following commands to run the test:
 
 1. `make setup_pg`
-2. `make start_pg`
```

Comment:
This is not managed by the RSpec test itself so that we can ensure that each test starts with the Postgres servers being in the same state.